src/model/Policy.lua

```diff
@@ -346,8 +381,11 @@ function Policy:removeFilteredPolicy(sec, ptype, fieldIndex, fieldValues)
             table.insert(tmp, rule)
         end
     end
-
+    self.model[sec][ptype].policyMap={}
     self.model[sec][ptype].policy = tmp
+    for i,policy in pairs(self.model[sec][ptype].policy) do
+        self.model[sec][ptype].policyMap[table.concat(policy,",")]=i
+    end
```

Comment:
The for loop refers to the `removePolicy()` above .

Reply:
Got it.

Reply:
Got it.